Tests/AK/TestMemoryStream.cpp

```diff
@@ -8,6 +8,10 @@
 #include <AK/String.h>
 #include <LibTest/TestCase.h>
 
+// Note: This is knowledge about the internals of AllocatingMemoryStream, which we use to write better test cases.
+//       If this doesn't match the actual chunk size, test cases will not be able to test the edge cases they want to test.
+static constexpr size_t chunk_size = 4096;
```

Comment:
Maybe expose the constant as `AllocatingMemoryStream::CHUNK_SIZE` then? It's not exactly a secret :^)

Reply:
Good idea, will do.